source/fundamentals/logging.txt

```diff
@@ -18,6 +25,15 @@ Starting in version 2.18, the {+driver-short+} uses the standard
 In this guide, you can learn how to use the driver to configure logging for your 
 application. 
 
+.. important::
+
+   To use this feature, you must include the `Microsoft.Extensions.Logging
+   namespace <https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging>`__:
+
+   .. code-block:: csharp
+
+      using Microsoft.Extensions.Logging;
```

Comment:
I: Microsoft.Extensions.Logging is a package you have to add to your project. The `using` statement simplifies using an already added package. (easy mistake to make if you don't use .net much!)

```suggestion
   To use this feature, you must add the `Microsoft.Extensions.Logging
   package <https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging>`__
   to your project.
```